.github/workflows/auto-release.yml

```diff
@@ -0,0 +1,13 @@
+name: 'auto release'
```

Comment:
Thanks for adding this! FYI don't forget to add the `GH_SECRET` when you are adding this workflow into a package pipeline. 

I just added it for this repo, but it will need to be manually added in the repo settings each time we add this action.

Reply:
doh right, thanks for adding it